.travis.yml

```diff
@@ -123,16 +120,16 @@ jobs:
         - |
           export CONDUIT_TAG=$(. bin/_tag.sh ; clean_head_root_tag)
           echo "CONDUIT_TAG=${CONDUIT_TAG}"
-        - export PROXY_RELEASE=1 BUILD_DEBUG=1 DOCKER_TRACE=1
+        - export BUILD_DEBUG=1 DOCKER_TRACE=1
 
       script:
-        - bin/docker-build
+        - SKIP_TESTS=1 bin/docker-build
 
       after_success:
         - bin/docker-push-deps
         - bin/docker-push $CONDUIT_TAG
         - bin/docker-retag-all $CONDUIT_TAG master && bin/docker-push master
-        - target/cli/linux/conduit install --conduit-version=$CONDUIT_TAG |tee conduit.yml
+        - target/cli/linux/conduit install --version=$CONDUIT_TAG |tee conduit.yml
```

Comment:
This looks like a bad merge; didn't we just rename `--version` to `--conduit-version`?